### PR TITLE
4 get api reviews

### PR DESCRIPTION
**Pull request by nasben64**

### CODE REVIEW

File: error-handler/app-errors.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    status: err.status || 500,
+    msg: err.message,
+  });
+};
+
+// this error handler will be called as the last in
+// the app.js to catch any an unexpected errors including
+// server errors
+exports.catchAllErrors = (err, req, res, next) => {
+  console.log(err, "error coming from app.use");
```
Comment (hevdevs) [inline error-handler/app-errors.js]: remove console.log before merge

File: controllers/reviews.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { selectAllReviews } = require("../models/reviews");
+
+exports.getAllReviews = (req, res, next) => {
+  selectAllReviews()
+    .then((reviews) => {
+      res.status(200).send(reviews);
```
Comment (hevdevs) [inline controllers/reviews.js]: we always want to be sending our response back as an object with a descriptive key in our controller i.e. .send({reviews: response}) - your tests may need a slight refactor to compensate

**Review by hevdevs — APPROVED** [review]
Just a couple of minor changes to make here - make sure to remove commented out code too before merge. Feel free to merge without another PR once done!

File: error-handler/app-errors.js (hunk 1 above)
Comment (nasben64) [reply error-handler/app-errors.js]: this catch all error message was recommended today by Liam to catch any unhandled errors, but I can remove it if you think is extra code.

File: controllers/reviews.js (hunk 2 above)
Comment (nasben64) [reply controllers/reviews.js]: ok I will change this

**Pull request merged**